### DISCUSSION TITLE
❌ [RFC] Space support `block` & `noStyle`

### DISCUSSION THREAD

**Discussion by zombieJ** [ant-design/ant-design#43935] [RFCs - archive]
# 📋 RFC

## Summary

Space support `block` prop to enable display as `flex` instead of `inline-flex`. `noStyle` to not to wrap `space-item` dom on the children.

## Basic example

```tsx
<Space block noStyle direction="vertical">
  <div>abc</div>
  <div>abc</div>
  <div>abc</div>
</Space>
```

## Motivation

Grid component is heavy in some case when we only need simplify flex view. Current Space is always wrap with `space-item` which can not easy to do some flex style like stretch align.

## Detailed

Support:
* block: to display as `flex`
* noStyle: remove wrap item element

## Drawbacks

will be similar to Grid

## Adoption strategy

Apply new component in minor version.

**Comment by zombieJ** [comment]
Hold till vote. @ant-design/ant-design-collaborators

**Comment by crazyair** [comment]
`noStyle`? 是否可以用`itemRender =(dom) => dom`
